dataactbroker/helpers/pandas_helper.py

```diff
@@ -6,22 +6,24 @@
 logger = logging.getLogger(__name__)
 
 
-def check_dataframe_diff(new_data, model, id_col, sort_cols, lambda_funcs=None):
+def check_dataframe_diff(new_data, model, sort_cols, del_cols=None, lambda_funcs=None):
```

Comment:
Since the ID wasn't optional, I don't think this array should be either. It should contain at the very least the non-optional ID column. If there's a super weird exception at some point the value passed can just be an empty array